sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/SingleFileReportImporter.java

```diff
@@ -20,58 +20,80 @@
 package org.sonar.plugins.php.phpunit;
 
 import java.io.File;
-import java.util.Optional;
+import java.io.IOException;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 import org.sonar.api.batch.sensor.SensorContext;
 import org.sonar.api.utils.log.Logger;
-import org.sonar.api.utils.log.Loggers;
+import org.sonar.plugins.php.warning.AnalysisWarningsWrapper;
+import org.sonarsource.analyzer.commons.ExternalReportProvider;
+import org.sonarsource.analyzer.commons.xml.ParseException;
 
 public abstract class SingleFileReportImporter implements ReportImporter {
```

Comment:
Since this now also handles importing multiple report files, and there is no `MultiPathImporter` anymore, what do you think of calling this class `ReportImporter` or `ReportFilesImporter`? I think `SingleFileReportImporter` is not appropriate anymore.